zerver/lib/actions.py

```diff
@@ -768,6 +822,7 @@ def recipient_for_emails(emails, not_forged_mirror_message,
         return get_recipient(Recipient.PERSONAL, list(recipient_profile_ids)[0])
 
 def already_sent_mirrored_message_id(message):
+    # type: (Message) -> int
```

Comment:
Yes. Actually not using `None` also works, so I didn't care about using `Optional`. But later I realized that it'll be better to use `Optional`. So I scanned all my annotations and put `Optional` wherever required. But it seems I missed out some places.